MoEngageInApps.podspec

```diff
@@ -0,0 +1,23 @@
+
+Pod::Spec.new do |s|
+  require_relative 'Utils/spec'
+  s.extend MoEngageSDK::Spec
+  s.define
+
+  s.version      = '6.00.3'
+  s.summary      =  "#{s.name} Campaigns for iOS"
+  s.description  = <<-DESC
+                     MoEngageInApp Messaging are custom views which you can send to a segment of users to show custom messages or give new offers or take to some specific pages.
+                     They can be created from your MoEngage dashboard and MOInApp framework enables you to show those InApp campaigns in your iOS Apps.
```

Comment:
MoEngageInApp is the framework name I guess, not MOInApp

Reply:
Updated name